.ci_support/compute_build_graph.py

```diff
@@ -353,7 +353,9 @@ def collapse_subpackage_nodes(graph):
             if subpackages:
                 remap_edges = [edge for edge in graph.edges() if edge[1] in subpackages]
                 for edge in remap_edges:
-                    graph.add_edge(edge[0], master_key)
+                    # make sure not to add references to yourself
+                    if edge[0] != master_key:
+                        graph.add_edge(edge[0], master_key)
```

Comment:
@isuruf @conda-forge/core This is a bug fix for build recipes with multiple outputs that have a specific dependency order.